src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -462,5 +544,30 @@ private static int GetTypeDistance(string typea, string typeb, ProtoCore.Core co
             }
 
         }
+
+        /// <summary>
+        /// The No Recommendations or Low Confidence Title
+        /// </summary>
+        public string NoRecommendationsOrLowConfidenceTitle { get; set; }
+
+        /// <summary>
+        /// The No Recommendations or Low Confidence message
+        /// </summary>
+        public string NoRecommendationsOrLowConfidenceMessage { get; set; }
+
+        /// <summary>
+        /// The Low Confidence additonal message
+        /// </summary>
+        public string LowConfidenceMessageAdditional { get; set; }
+
+        /// <summary>
+        /// Indicates if display the No recommendations / Low confidence message (image and texts)
+        /// </summary>
+        public bool DisplayNoRecommendationsLowConfidence { get; set; }
+
+        /// <summary>
+        /// Indicates if display the Low confidence option and Tooltip
+        /// </summary>
+        public bool DisplayLowConfidence { get; set; }
```

Comment:
Do we need all of these properties? It seems most of them will have same/opposite value of `Models.NodeAutocompleteSuggestion.MLRecommendation`?

Reply:
Yes, according to the Mockups we have different messages and behavior based on the type of the results: by use, recommendations and confidence score.